.github/workflows/tests.yml

```diff
@@ -66,10 +66,10 @@ jobs:
       REDIS_QUEUE_HOST: redis
       REDIS_STATE_HOST: redis
       ASAN_OPTIONS: "verbosity=1:halt_on_error=1"
-      TSAN_OPTIONS: "verbosity=1:halt_on_error=1:suppressions=/code/faabric/thread-sanitizer-ignorelist.txt:history_size=7"
+      TSAN_OPTIONS: "verbosity=1 halt_on_error=1 suppressions=./thread-sanitizer-ignorelist.txt history_size=4"
```

Comment:
We need this path to be relative to the working directory, to allow the tests to run on any checked-out path (i.e. GHA with `actions/checkout`).